README.md

```diff
@@ -17,11 +17,18 @@ Many other things are rumored to be working and have drivers, but I wasn't able
 
 ## Build
 
-The default setting is to cross-compile from x86_64 to aarch64, if your build system is not `x86_64-linux` you have to modify `buildSystem` in `flake.nix`.
+There are two main ways to build the ISO:
+
+1. The default setting is to cross-compile from x86_64 to aarch64. This takes several hours as all packages are compiled from scratch.
+2. Use WSL on the Lenovo Yoga Slim 7x to build using Nix. This generally takes 25 minutes as only the kernel is compiled from scratch.
+
+if your build system is not `x86_64-linux` you have to modify `buildSystem` in `flake.nix`, e.g. to `aarch64-linux` if building in WSL.
+
+It's recommended to build using WSL, where you can install Nix by install e.g. Ubuntu in WSL and then installing Nix (shell) as usual [following the guide](https://nixos.org/download/).
```

Comment:
I am not comfortable saying that "It's recommended to build using WSL", mainly because I only test the cross-compile method, and I can't vouch for anything using WSL.

Also I think if we can we should link to some WSL guides, e.g. https://github.com/nix-community/NixOS-WSL that you already mentioned in the issue, and maybe something for Ubuntu as well.

Reply:
This should be fixed now